user-guide/modules/ROOT/pages/release-process.adoc

```diff
@@ -1,69 +1,33 @@
 = Release Process
 
-@anarthal: shorten this and phrase it from a user perspective. Keep the long version for the contributor guide and place a link here
-
-The Boost libraries are released publicly three times per year:
+Boost libraries are released publicly three times per year:
 
 [circle]
 * 2nd April
 * 2nd August
 * 2nd December
 
-Each release will contain updates to existing libraries, and some releases will contain new libraries. The release is built from the *master* branch of Boost's GitHub site: https://github.com/cppalliance.
+Each release will contain updates to existing libraries, and some releases will contain new libraries. The release is built from the Boost GitHub site: https://github.com/cppalliance.
 
 == Preparing for a Boost Release
```

Comment:
I'm not sure about this - a user doesn't prepare for a Boost Release, a contributor does. I think the ordering is a little bit upside down from a user perspective.

I'd first explain that *all* Boost libraries are released together (this is not the usual thing).
Then would jump to explain that we've got a strict policy on how to merge new code when the release approaches, restricting increasingly what can be merged as the release deadline approaches, to guarantee stability.
Then add the regression test & beta paragraphs which are nice.

Reply:
Yes. The content about the release process should be split into the user and contributor guides as we touch it.